### ISSUE TITLE
Video stream not working after changing Tello IP Address

### ISSUE THREAD

**Issue by SMRazaRizvi96** [appie-17/tello_driver#12]
I connected my Tello with my own WiFi connection and changed the Tello's IP Address everywhere inside the codes.

After running the tello_driver_node.py through the tello_node.launch, the code successfully connects to the Tello but the video stream gives an error.

Tello: 16:37:38.797:  Info: start video (cmd=0x25 seq=0x01e4)
Tello: 16:37:39.799: Error: video recv: timeout
Tello: 16:37:39.800:  Info: start video (cmd=0x25 seq=0x01e4)
Tello: 16:37:40.803: Error: video recv: timeout
Tello: 16:37:40.803:  Info: start video (cmd=0x25 seq=0x01e4)
Tello: 16:37:41.807: Error: video recv: timeout
Tello: 16:37:41.807:  Info: start video (cmd=0x25 seq=0x01e4)

How to solve this?
![video_not_working](https://user-images.githubusercontent.com/74411560/132530674-59e515da-7e81-4e18-8a2a-d233321db5b8.png)

**Comment by Helge543** [comment]
Hi, are you using a virtual machine? I ran into the same issue some time ago when I used a VM with Ubuntu 18.04 on it and ROS Melodic. When I switched to Ubuntu 20.04 with ROS Noetic (without using a VM) I could fix this. If this is not your case, pls read through the Issues [here ](https://github.com/hanyazou/TelloPy/issues)on github. It seems that many people have problems with that. You also could try a firmware update via tello App. Hope it helped.

**Comment by SMRazaRizvi96** [comment]
Yes I am using a VM, but this problem occurs only when I connect my Tello to my home WiFi.
Otherwise, using Tello's own WiFI, I am able to get the video stream.

Is it the case that we cannot get the Tello's stream when we connect it to any other WiFi?
If not, if I switch to Ubuntu without a VM and connect my Tello to my home WiFi, will I be able to get the video stream?

**Comment by Helge543** [comment]
okay, I saw that [these ](https://www.hajim.rochester.edu/senior-design-day/architecture-for-a-cooperative-slam-for-multi-uav-systems/) guys also had problems with the connection when using a VM. Read the article from "Archictecture". They were overcoming this issue by using a USB wireless antenna before they switched to native Linux machines. So, you also could try that.

**Comment by SMRazaRizvi96** [comment]
So summarizing the discussion, you are suggesting that I try using a USB wireless antenna to connect my laptop to my home WiFi, and also connect my Tello to my Home Wifi, and then the video stream should work, right?

**Comment by Helge543** [comment]
Just give it a try. And if it works, pls post it here because it may help other people.

**Comment by SMRazaRizvi96** [comment]
I didn't try this.
For what I wanted to do, I kept using Tello's WiFi for the video stream and found workaround for the other tasks.

Thanks anyways.

**Comment by Helge543** [comment]
okay. no problem, you're welcome
